framework/core/js/src/common/components/SelectDropdown.tsx

```diff
@@ -50,7 +49,8 @@ export default class SelectDropdown<CustomAttrs extends ISelectDropdownAttrs = I
     const activeChild = children.find(isActive);
     let label = (activeChild && typeof activeChild === 'object' && 'children' in activeChild && activeChild.children) || this.attrs.defaultLabel;
 
-    label = extractText(label);
+    // @ts-ignore
+    if (label instanceof Array) label = label[0];
```

Comment:
```suggestion
    if (Array.isArray(label)) label = label[0];
```

Reply:
Whilst I don't mind making this change, the advice from @SychO9 was to revert to the previous code for safety (https://github.com/flarum/framework/pull/3608). 

Any comments/thoughts/suggestions welcome :)

Reply:
I don't mind this change either